packages/node/src/app/analytics-node.ts

```diff
@@ -33,25 +33,26 @@ export class Analytics extends NodeEmitter implements CoreAnalytics {
 
   constructor(settings: AnalyticsSettings) {
     super()
+    settings.flushAt ??= settings.maxEventsInBatch ??= 15
+    settings.maxRetries ??= 3
+    settings.flushInterval ??= 10000
```

Comment:
Do these `??=` get transpiled? Basically just want to make sure this doesn't cause any problems with the minimum node.js versions we support.

Reply:
Yes they get transpired. We have tests that target our minimum node version, so not worried.

Reply:
Sweet, thanks!